scripts/publish-wasmtime.sh

```diff
@@ -2,12 +2,19 @@
 set -euo pipefail
 
 # This is a convenience script for maintainers publishing a new version of
-# Wasmtime to crates.io. To use, bump the version number below, run the
-# script, and then run the commands that the script prints.
+# Wasmtime to crates.io. To use, first bump the Wasmtieme versions with
+# `scripts/bump-wasmtime-version.sh` and Cranelifte versions with
+# `scripts/bump-cranelift-version.sh`, then run this script, and run the
+# commands that it prints.
+#
+# Don't forget to push a git tag for this release!
```

Comment:
May I suggest to actually print this message? It's likely users of this script won't look at its source when using it.